pkg/cmd/step/create/step_create_task.go

```diff
@@ -427,6 +425,7 @@ func (o *StepCreateTaskOptions) GenerateTektonCRDs(packsDir string, projectConfi
 	if projectConfig.NoReleasePrepare {
 		o.NoReleasePrepare = true
 	}
+	pipelineConfig := updatedProjectConfig.PipelineConfig
 	err = o.setVersionOnReleasePipelines(pipelineConfig)
```

Comment:
So really, this is where the rub was, right? It passed the original /stale config to `setVersionOnReleasePipelines`.

Reply:
Yup, exactly.